### DISCUSSION TITLE
AV esset block sir

### DISCUSSION THREAD

**Discussion by notfont404** [t3l3machus/Villain#7] [Q&A]
AV esset block sir

**Comment by t3l3machus** [comment]
@notfont404 so ESET AV catches Villain paylaods? Did you use "obfuscate"? if yes, did you try multiple times? I am asking because the auto-obfuscate creates random variations, maybe one could actually work. Can you share any screenshots? thanks

**Comment by Cruzzop** [comment]
Sir please add Android OS also🙏🏻

**Reply by carlos837** [reply]
As a payload or getting a shell from Android?

**Reply by Cruzzop** [reply]
> As a payload or getting a shell from Android?

Are you try this in Android before?????

**Reply by carlos837** [reply]
Nope, but I think I could

**Reply by carlos837** [reply]
I think Termux could handle this

**Reply by carlos837** [reply]
Im trying it, gimme a sec

**Reply by Cruzzop** [reply]
> Im trying it, gimme a sec

Okk after trying please tell me the process

**Reply by carlos837** [reply]
I can't do it, I can't install packages on termux

**Reply by Cruzzop** [reply]
> I can't do it, I can't install packages on termux

Okkk friend

**Comment by aksrivastava** [comment]
**try this to bypass the defender with villain**
https://youtu.be/EZOW40S_cTM
